llvm/lib/Target/RISCV/RISCVInstrInfoZvk.td

```diff
@@ -57,8 +55,7 @@ multiclass VROR_IV_V_X_I<string opcodestr, bits<6> funct6>
   def I : RVInstIVI_VROR<funct6, (outs VR:$vd),
               (ins VR:$vs2, uimm6:$imm, VMaskOp:$vm),
               opcodestr # ".vi", "$vd, $vs2, $imm$vm">,
-         Sched<[WriteVIALUI_WorstCase, ReadVIALUV_WorstCase,
-                ReadVMask]>;
+          SchedUnaryMC<"WriteVROR", "ReadVROR">;
```

Comment:
Established convention is a different Write based on the operand type. See `WriteVIALUV`, `WriteVIALUX`, and `WriteVIALUI`